sources/ru/jaomix.py

```diff
@@ -78,10 +47,58 @@ def parse_chapters(self, page_soup):
                     "id": chap_id,
                     "volume": vol_id,
                     "title": a.text.strip(),
-                    "url": self.home_url.rstrip("/") + a["href"],
+                    "url": self.absolute_url(a['href']),
                 }
             )
 
+        # self.novel_id = soup.select_one("div.like-but")["id"]
```

Comment:
you don't have to keep the previous code in comments. these may never be used in future.